docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,179 @@
+[role="xpack"]
+[[cases-webhook-action-type]]
+=== Webhook - Case Management connector and action
+++++
+<titleabbrev>Webhook - Case Management</titleabbrev>
+++++
+
+The Webhook - Case Management connector uses https://github.com/axios/axios[axios] to send POST, PUT, and GET requests to a Case Management RESTful API web service. The resulting connector can be used with Kibana Cases to create and update a case in the third party web service.
+
+[float]
+[[webhook-connector-configuration]]
+==== Connector configuration
+
+Webhook - Case Management connectors have the following configuration properties.
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+Require authentication:: If true, a username and password for login type authentication must be provided.
+Username::      Username for HTTP basic authentication.
+Password::  Password for HTTP basic authentication.
+Headers::   A set of key-value pairs sent as headers with the request
+Create Case Method::  REST API HTTP request method to create case in third party system, either `post`(default), `put`, or `patch`.
+Create Case URL::       Rest API URL to create case in third party system. If you are using the <<action-settings, `xpack.actions.allowedHosts`>> setting, make sure the hostname is added to the allowed hosts.
+Create Case Object::      A JSON payload sent to the create case URL to create case. Use the variable selector to add Kibana Cases data to the payload. Required variables are `case.title` and `case.description`. For example:
++
+[source,text]
+--
+{
+	"fields": {
+	  "summary": {{{case.title}}},
+	  "description": {{{case.description}}},
+	  "labels": {{{case.tags}}}
+	}
+}
+--
+Note: Due to Mustache template variables (the text enclosed in triple braces, for example, `{{{case.title}}}`), the JSON is not validated in this step. The JSON is validated once the mustache variables have been placed when REST method is executed. Manually ensure that the JSON is valid besides the mustache variables so the later validation will pass.
+
+Create Case Response - Case ID Key::   JSON key in create case response that contains the external case id
```

Comment:
I suggest to have a working example with the real configuration to help the user to understand better the capabilities of the new connector.
It could be some existing integration like ServiceNow or Jira. The service endpoint won't be the real thing, but all the parameters and requests bodies will be exactly what the real integration have. Only thin the user will need to do is to follow the suggested steps but replace the service endpoints with their own.

Reply:
i kind of followed the other documentation. should i do this in a new section, or break from the model of other docs and do it inline?

Reply:
You are right, the current doc follows the structure of the other connectors. But having the section example looks like a great idea!
Also, @elastic/security-docs maybe you can suggest something here?

Reply:
I've attempted an example in my [last commit](https://github.com/elastic/kibana/pull/137726/commits/214745a51158e11eec955370fed854dd00ead01b)